### ISSUE TITLE
v1.3.0 - missing schedule icon from notebook

### ISSUE THREAD

**Issue by rubenvarela** [jupyter-server/jupyter-scheduler#346]
## Description

On release v1.3.0, inside of a notebook, on the icon bar, the schedule icon is missing.

![Screenshot 2023-02-26 030635](https://user-images.githubusercontent.com/3499097/221399435-e135290e-9d24-4f76-ab67-b0cf230b2afa.png)

## Reproduce

1. Install v1.3.0 via pip on a non-conda environment
2. run `jupyter-lab`
3. open a notebook and you'll see the icon missing.

## Expected behavior

The schedule icon should show,

![Screenshot 2023-02-26 031231](https://user-images.githubusercontent.com/3499097/221399530-b028ac7f-9f73-4f7b-8776-adfb4318e7bf.png)

## Context

- Operating System and version: 

```
lsb_release -a
No LSB modules are available.
Distributor ID: Debian
Description:    Debian GNU/Linux 11 (bullseye)
Release:        11
Codename:       bullseye
```

- Browser and version: Firefox 110

The issue comes from changes to 8f697f1acaa070f9252de7e98f2215209f51a780 

Reverting that commit locally, then `jlpm build` brings the icon back for me.

**Comment by berl** [comment]
I confirm this @rubenvarela. 

as someone trying out `jupyter-scheduler` for the first time tonight, this bug gives the sense that nothing is working despite following the install instructions. Rolling back to 1.2.0 worked as a fix.

**Comment by dlqqq** [comment]
Investigating this issue now.

**Comment by 3coins** [comment]
@rubenvarela 
Can you try changes on this PR. https://github.com/jupyter-server/jupyter-scheduler/pull/347

**Comment by rubenvarela** [comment]
@3coins the changes on #347 do bring back the icon!

**Comment by dlqqq** [comment]
Very sorry about this oversight. Thank you for the quick reporting! This should be fixed in 1.3.1.

**Comment by rubenvarela** [comment]
@dlqqq no problem, it happens. 

Appreciate you all taking a look at it!
